services/api/src/resources/task/task_definition_resolvers.ts

```diff
@@ -142,18 +142,31 @@ export const resolveTasksForEnvironment = async (
     Sql.selectAdvancedTaskDefinitionsForGroups(projectGroupsFiltered)
   );
 
+  const systemWideRows = await query(
+    sqlClientPool,
+    Sql.selectAdvancedTaskDefinitionsForSystem()
+  );
+
+
+
```

Comment:
just small nits, fair amount of additional lines throughout that bloat the diff a bit